src/layouts/BasicLayout.js

```diff
@@ -87,27 +82,52 @@ class BasicLayout extends React.PureComponent {
 
     this.state = {
       sessionExitModalVisible: false,
+      breadcrumb: {},
+      showLoginBanner: true,
     };
   }
 
-  getChildContext() {
-    const { location } = this.props;
-    const { route } = this.props;
-    return {
-      location,
-      breadcrumbNameMap: this.breadcrumbNameMap,
-      routes: route.routes,
-    };
+  componentDidMount() {
+    const {
+      location: { pathname },
+    } = this.props;
+    let exactPath = {};
```

Comment:
We should consider creating a function for the logic on lines 94 - 102 to avoid repeating duplicate functionality (DRY principle).